t/cli/test_prometheus_stream.sh

```diff
@@ -60,7 +60,7 @@ sleep 1 # wait for sync
 
 out="$(curl http://127.0.0.1:9091/apisix/prometheus/metrics)"
 if ! echo "$out" | grep "apisix_stream_connection_total{route=\"1\"} 1" > /dev/null; then
-    echo "failed: prometheus can't work in stream subsystem"
+    echo "failed: prometheus can't work in both http & stream subsystem"
```

Comment:
Additional testing is needed; the existing tests cannot verify whether prometheus-cache is correctly generated when use_apisix_base=false.

Reply:
I didn't find out how to set use_apisix_base=false in ut environment, it's setted in env.lua:
```
    local use_apisix_base = true
    if not or_info:find("apisix-nginx-module", 1, true) then
        use_apisix_base = false
    end
```